src/Control/Director.php

```diff
@@ -76,40 +80,47 @@ class Director implements TemplateGlobalProvider
     private static $alternate_base_folder;
```

Comment:
folders are urls are different though. We need to set a base folder when mocking a subdirectory, which doesn't mean we want to change the base url.